devel/fnc/portfile

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+
+name            fnc
+version         0.6
+distname        fnc-${version}
+categories      devel
+supported_archs x86_64 i386 arm64
```

Comment:
I think you should be OK to remove `supported_archs`.  No need to specify it unless the port actually has architecture restrictions.